app/validators/claim/base_claim_validator.rb

```diff
@@ -177,13 +174,11 @@ def validate_trial_fixed_notice_at
   # cannot be more than 5 years old
   # cannot be before trial_fixed_notice_at
   def validate_trial_fixed_at
-    return if @record.disable_for_state_transition.eql?(:only_amount_assessed)
-    if @record.case_type && @record.requires_cracked_dates?
-      validate_presence(:trial_fixed_at, 'blank')
-      validate_too_far_in_past(:trial_fixed_at)
-      validate_on_or_after(@record.trial_fixed_notice_at, :trial_fixed_at,
-                           'check_not_earlier_than_trial_fixed_notice_at')
-    end
+    return if ignore_validation_for_trial_dates
```

Comment:
naming suggestion: would `ignore_validation_for_cracked_trial_dates` be more applicable and descriptive -